financial-analyst-deepseek/financial-analyst-langgraph/finance_langgraph.py

```
@@ -0,0 +1,103 @@
+from langgraph.graph import StateGraph, END ,START
+from pydantic import BaseModel, Field
+from langchain_core.runnables import Runnable
```

Ruff (0.11.9)

`langchain_core.runnables.Runnable` imported but unused

Remove unused import: `langchain_core.runnables.Runnable`

(F401)